lib/bots/tobybot.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ Copyright 2016-2017 Resin.io
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+// TobyBot listens for Github Issues, and counts the palindromes in them
+//
+// To set up, follow the main guide, and define these respective env vars:
+// TOBYBOT_WEBHOOK_SECRET
+// TOBYBOT_INTEGRATION_ID
+// TOBYBOT_PEM
+// TOBYBOT_NAME
+// TOBYBOT_EMAIL
```

Comment:
@hedss I guess that's a holdover from the syncbot example and the email setting is not really needed, right? Or I'm double-missing something?